### PR TITLE
Review 1

### PR DESCRIPTION
**Pull request by rhs0266**

### CODE REVIEW

File: ItemCatalogTest/test_item_catalog.py [hunk 1]
```diff
@@ -19,110 +19,179 @@
...
 #
 # Helper functions
 #
 #
 @pytest.fixture()
 def df():
-    filename = 'item_catalog.tsv'
-    df = read_csv(filename, sep='\t', encoding='utf8')
+    filename = "item_catalog.tsv"
+    df = read_csv(filename, sep="\t", encoding="utf8")
```
Comment (rhs0266) [inline ItemCatalogTest/test_item_catalog.py:32]: Some companies are using `csv` extension. May need to support it.

File: ItemCatalogTest/test_item_catalog.py [hunk 2]
```diff
@@ -19,110 +20,213 @@
...
+        "review_count",
+        "rating",
+        "shipping_charge",
+        "brand",
+        "longtail_yn",  # Not exist in our feed file. Could you explain more detail?
+        "updated_at",  # RMP uses `update_time` for this field.
+        "availability",
+        "blocked",
+    }
+    dt.validate(df.columns, doc_based_required_names)
```
Comment (rhs0266) [inline ItemCatalogTest/test_item_catalog.py:163]: QQ: The `validate` function checks the columns are strictly equal to our columns. What if the feed file doesn't? Actually, none of our current clients include the whole field.

**Review by rhs0266 — COMMENTED** [review]
Some columns may not exist in the file. In this case, `row.some_column` will raise an exception. For example, an optional field named "brand" is optional, so the company do not make that column at all. In this case, `test_column_brand` will raise error even if the field is optional.

Some fields are not handled. For example, `size`, `material`, `pattern`, etc.

**Review by jongnam — COMMENTED** [review]
Please see inline comments.

File: ItemCatalogTest/test_item_catalog.py [hunk 3]
```diff
@@ -195,44 +274,52 @@ def assert_wellformed_price_pc(index, row):
     print_message(test_passed, index, row, assert_wellformed_price_pc.__name__)
     assert test_passed
```
[the remark this answers is not in the document]
Comment (rhs0266) [reply ItemCatalogTest/test_item_catalog.py:305]: They are not following our latest convention. I guess we exceptionally handle that case.

File: ItemCatalogTest/test_item_catalog.py [hunk 4]
```diff
@@ -19,110 +19,179 @@
...
-        'rating',
-        'shipping',
-        'brand',
-        'longtail_yn',
-        'updated_at',
-        'availability',
-        'blocked'}
+        "id",
+        "seller_id",
+        "seller_name",  # RMP uses `seller_title` for this field.
```
[the remark this answers is not in the document]
Comment (yunsang-hwang-moloco) [reply ItemCatalogTest/test_item_catalog.py]: `seller_title` is correct. Will guide further customers to use `seller_title`.

File: ItemCatalogTest/test_item_catalog.py [hunk 5]
```diff
@@ -19,110 +19,179 @@
...
+        "normal_price",
+        "price_pc",
+        "link",
+        "image_link",
+        "category",
+        "review_count",
+        "rating",
+        "shipping",
+        "brand",
+        "longtail_yn",  # Not exist in our feed file. Could you explain more detail?
```
[the remark this answers is not in the document]
Comment (yunsang-hwang-moloco) [reply ItemCatalogTest/test_item_catalog.py:158]: `longtail_yn` is not universal. Can be removed.

File: ItemCatalogTest/test_item_catalog.py (hunk 1 above)
Comment (jongnam) [reply ItemCatalogTest/test_item_catalog.py:32]: We will use CSV as our standard format.

**Pull request closed without merging**